apps/sms/views/inbox/js/inbox.js

```diff
@@ -983,6 +975,8 @@ var InboxView = {
   onDraftSaved: function inbox_onDraftSaved(draft) {
     var threadToUpdate = draft.threadId ? Threads.get(draft.threadId) : draft;
     this.updateThread(threadToUpdate);
+
+    this.showDraftSavedBanner();
```

Comment:
what we _really_ should do when the user swipes left is wait for visibiilitychange event with document.hidden == false to show a banner :) But this can be in another bug.